packages/esm-commons-lib/package-lock.json

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
This seems to be a mistake. We're using yarn for package management and we already have a `yarn.lock`

Reply:
Nice catch